src/main/java/uk/ac/diamond/ispyb/api/DataCollection.java

```diff
@@ -0,0 +1,593 @@
+package uk.ac.diamond.ispyb.api;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.sql.Timestamp;
+
+public class DataCollection {
+    private Long Id;
+    private Long GroupId;
+    private Long dcNumber;
+    private Long apertureId;
+    private Long blSubSample;
+    private Long detectorId;
+    private Double sampleId;
+    private Long polarisationId;
```

Comment:
There is a polarisation (float) column in the DataCollection table, but no ploarisationId. In any case, neither are returned by the retrieve_dc sproc, so I don't think we need this.

Reply:
Ah, polarisationId and polarisation are fields in the dna.xml object as are a few others which aren't in the sproc (The full list is: sampleId, polarisationId, positionId, actualCenteringPosition, beamShape, polarisation, printableForReport, screeningOrigId)

Reply:
I can either delete these fields or add them to the stored procedure.

Reply:
Hmm, `sampleId` would be useful (or `blSampleId` would be better since we already have `blSubSampleId`). 

If you're able to add them to the sproc (requires another PR obviously), that's cool, if not just delete them for now and we'll do it in the next release.

BTW: `blSubSample` should be `blSubSampleId`, and the datatype for `sampleId` should be `Long`.

Reply:
Should be pretty quick, I'll get on that now.